websites/P/Pixelmon Mod/presence.ts

```diff
@@ -0,0 +1,63 @@
+const presence = new Presence({
+		clientId: "976435781486911509"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	page = document.location.href;
+
+let title: Element, title2: Element;
```

Comment:
Please type these properly if you really wanna declare them outside of the event, otherwise just make them constants and use them only where needed